setup_dev_env.sh

```diff
@@ -117,6 +117,10 @@ if [ $ERR -ne 0 ]; then
     exit $ERR
 fi
 
+DOCKER_GID=$(id -g $USER)
+if [ -z "$DOCKER_GID" ]; then
```

Comment:
Under what circumstances would `id -g $USER` return nothing? Is setting the GID to the UID the right thing to do?

AFAIK, they are different ID spaces.